lib/taurus/qt/qtcore/util/emitter.py

```diff
@@ -252,19 +303,24 @@ def next(self):
                 Qt.QApplication.instance().restoreOverrideCursor()
                 self._cursor = False
 
-        except Queue.Empty:
+        except Empty:
             self.log.warning(traceback.format_exc())
             pass
         except:
             self.log.warning(traceback.format_exc())
         return
 
     def run(self):
-        Qt.QApplication.instance().thread().sleep(int(self.timewait / 1000)
-                                                  if self.timewait > 10 else int(self.timewait))  # wait(self.sleep)
+        Qt.QApplication.instance().thread().sleep(
+            int(self.timewait / 1000) if self.timewait > 10 
+                                else int(self.timewait))
```

Comment:
It is not critical, but the following would be a lot more readable (and in PEP8 spirit):

```python
_sleep = self.timewait
if self.timewait > 10:
    _sleep /= 1000
Qt.QApplication.instance().thread().sleep(int(_sleep))
```
(it is 4 lines instead of 3, but it is much easier to follow)

Reply:
I removed the entire condition, it was an old tau-compatibility check that is not used anymore.